pkg/apis/apps/v1beta2/nodepool_conversion.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2024 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta2
+
+/*
+Implementing the hub method is pretty easy -- we just have to add an empty
+method called `Hub()` to serve as a
+[marker](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/conversion?tab=doc#Hub).
+*/
+
+// Hub marks this type as a conversion hub.
+func (*NodePool) Hub() {}
```

Comment:
We need improve v1beta1.NodePool conversion, because v1beta2.NodePool will be used as storage. this means that `v1beta1.NodePool` should implement `ConvertTo` and `ConvertFrom` function for converting between v1beta1 and v1beta2.